my-file.txt

```diff
@@ -0,0 +1,4 @@
+{"level":30,"time":1613691032156,"pid":13640,"hostname":"DESKTOP-AJNDNDT","msg":"Hi! Here is my first massage"}
```

Comment:
Delete this file

Reply:
change the filename of the logs file to miniurl.log (it will prevent future commits to this file since *.log files are ignored)

Reply:
how can I do this?
this is not in the Docker?